cmake/dca_testing.cmake

```diff
@@ -131,13 +118,13 @@ function(dca_add_gtest name)
 
     add_test(NAME ${name}
              COMMAND ${TEST_RUNNER} ${MPIEXEC_NUMPROC_FLAG} ${DCA_ADD_GTEST_MPI_NUMPROC}
-                     ${MPIEXEC_PREFLAGS} ${SMPIARGS_FLAG_MPI_CVD} ${CVD_LAUNCHER} "$<TARGET_FILE:${name}>")
+                     ${MPIEXEC_PREFLAGS} "$<TARGET_FILE:${name}>")
                  target_link_libraries(${name} ${MPI_C_LIBRARIES})
   else()
     if (TEST_RUNNER)
       add_test(NAME ${name}
                COMMAND ${TEST_RUNNER} ${MPIEXEC_NUMPROC_FLAG} 1
-	               ${MPIEXEC_PREFLAGS} ${SMPIARGS_FLAG_NOMPI} "$<TARGET_FILE:${name}>")
+	               ${MPIEXEC_PREFLAGS} "$<TARGET_FILE:${name}>")
```

Comment:
@PDoakORNL: I am not really aware of what the whole *_CVD flags where supposed to do or when they where introduced, so if I am missing something by removing them, please let me know.

Reply:
Readded the cvdlauncher and a cache flag that can point to it (to be set from the <system>.cmake file) I kept the CUDA_CVD option out of the test function as all the tests have the same requirements besides requiring CUDA and/or MPI.